src/toast/tests/spt3g.py

```diff
@@ -0,0 +1,406 @@
+
+        # The default export / import classes
+        meta_exporter = export_obs_meta(
+            noise_models=[
+                ("noise_model", "noise_model"),
+                ("el_weighted", "el_weighted"),
+            ]
+        )
+        data_exporter = export_obs_data(
+            shared_names=self.export_shared_names,
+            det_names=self.export_det_names,
+            interval_names=self.export_interval_names,
+            compress=False,
+        )
+
+        exporter = export_obs(
+            meta_export=meta_exporter,
+            data_export=data_exporter,
+        )
+
+        meta_importer = import_obs_meta(
+            noise_models=[
+                ("noise_model", "noise_model"),
+                ("el_weighted", "el_weighted"),
+            ]
+        )
+        data_importer = import_obs_data(
+            shared_names=self.import_shared_names,
+            det_names=self.import_det_names,
+            interval_names=self.import_interval_names,
+        )
+        importer = import_obs(
+            comm=data.comm.comm_group,
+            meta_import=meta_importer,
+            data_import=data_importer,
+        )
+
+        # Export the data, and make a copy for later comparison.
+        original = list()
+        g3data = list()
+        for ob in data.obs:
+            original.append(ob.duplicate(times="times"))
+            obframes = exporter(ob)
+            # There should be the original number of frame intervals plus
+            # one observation frame and one calibration frame
+            checktot = len(obframes) - 2
+            if ob.comm_row is not None:
+                checktot = ob.comm_row.allreduce(checktot, op=MPI.SUM)
+            if checktot != ob_n_frames[ob.name]:
+                msg = f"proc {data.comm.world_rank} on ob {ob.name} "
+                msg += f"has {checktot} scan frames, "
+                msg += f"not {ob_n_frames[ob.name]}"
+                print(msg, flush=True)
+                self.assertTrue(False)
+            g3data.append(obframes)
+
+        # Import the data
+        check_data = Data(comm=data.comm)
+
+        try:
+            for obframes in g3data:
+                check_data.obs.append(importer(obframes))
+
+            for ob in check_data.obs:
+                ob.redistribute(ob.comm_size, times="times")
+        except Exception as e:
+            import sys
+            import traceback
+
+            exc_type, exc_value, exc_traceback = sys.exc_info()
+            lines = traceback.format_exception(exc_type, exc_value, exc_traceback)
+            lines = [f"Proc {data.comm.world_rank}: {x}" for x in lines]
+            print("".join(lines), flush=True)
+
+        # Verify
+        for ob, orig in zip(check_data.obs, original):
+            if ob != orig:
+                print(f"-------- Proc {data.comm.world_rank} ---------\n{orig}\n{ob}")
+            self.assertTrue(ob == orig)
+
+    def test_save_load_nocomp(self):
```

Comment:
Even after reading the test source twice, I cannot figure out what "nocomp" means. "No computation" or "no comparison"? Could we add a comment?

Reply:
"no compression".  Will change it.